change/@fluentui-react-spinbutton-43864ed0-359e-407e-8250-36a85a8ba6b4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33143&commitId=898b363269fccb36b107fecc0179cba2835c536c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|9| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|9| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|6204| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>